e2e/test/scenarios/sharing/public.cy.spec.js

```diff
@@ -200,5 +201,48 @@ describe("scenarios > public", () => {
         });
       }),
     );
+
+    describe("Disable auto-apply filters", () => {
+      it("set auto-apply filters to false", () => {
+        visitDashboard(dashboardId);
+
+        openDashboardSidebar();
+        rightSidebar()
+          .findByLabelText("Auto-apply filters")
+          .click()
+          .should("not.be.checked");
+      });
+
+      Object.entries(USERS).map(([userType, setUser]) =>
```

Comment:
We have quite a lot of users to iterate over them in e2e like that. Rather than testing for all the users, is there anything specific you want to check for? To me it seems that this feature doesn't depend on user attributes/permissions so I'd just have a single test.

Reply:
Fixed by only running a single test now since it's a public dashboard so doesn't need to be tested by different users.